### ISSUE TITLE
Improving User Interface

### ISSUE THREAD

**Issue by mrEDitor** [TemporaryTeam/NoticEditor#48]
I suggest to discuss improving user interface
My version is:

![image](https://cloud.githubusercontent.com/assets/6552749/9273180/b923d596-4292-11e5-8a20-222f9ebc87f9.png)

* left empty space is structure tree.

**Add**-button also has options (main label should be "add notice"):

![image](https://cloud.githubusercontent.com/assets/6552749/9273262/75c37ae4-4293-11e5-871e-322eb123dbbf.png)

**"Search"** button showing corresponding tool as popup window.
**"Important"** can be ChoiceBox or SplitMenuBox, which can fast-switch between "Normal status" and "Important" while full list can also has additional statuses.
* middle empty space going to editing notice, adding files and perhaps, importing/exporting this particular notice;
* right empty space is preview.

Since panels are resizeable, it will be nice to add fast hiding to "View" menu:

![image](https://cloud.githubusercontent.com/assets/6552749/9273380/fb8e819a-4294-11e5-9078-38c532c3206d.png)

and some "Orientation" tool for changing orientation of editor and preview panels, likely with possibility to swap them.

What do you think?

**Comment by aNNiMON** [comment]
@mrEDitor it's great. I like this UI.

I would like to pay attention to Filter fields and Search button. While we type text, program performs instant search as is done now. If we push Search button, program performs full search in new window, how @NaikSoftware wants.

**Comment by NaikSoftware** [comment]
@mrEDitor , реализовал некоторые твои идеи у себя в ветке. Выглядит так.
![2015-08-14 18 39 36](https://cloud.githubusercontent.com/assets/4218994/9277693/7aa86fb2-42b4-11e5-8517-190bc134da54.png)

**Comment by mrEDitor** [comment]
Oh, sorry for double-reference. Didn't notice that commit message from my fork will popup here.

I am going to split controllers like:
![image](https://cloud.githubusercontent.com/assets/6552749/9410656/0b77d8a0-482a-11e5-979e-2f36085d56e3.png)
* yellow – Main.fxml + NoticeController: manage whole opened noticebook-file and settings
* purple – NoticeTreeView +...Contoller: manage noticebook structure
* green – NoticeView + ...Controller: manage single notice (editing, showing, editing plugins)
* red – AttachView + ...Controller
* blue – WebImport + ...Controller (move here from app menu since it imports/exports single notice)

Seems like blue tab should be splitted to separate tabs.

**Comment by NaikSoftware** [comment]
Придется все равно дергать методы друг у друга..

**Comment by mrEDitor** [comment]
А это, кстати говоря, лечится хорошей системой ивентов.

**Comment by mrEDitor** [comment]
Now I'm going to rework left panel and add to buttons icons from https://materialdesignicons.com/
![image](https://cloud.githubusercontent.com/assets/6552749/9423613/8a289b30-48d4-11e5-8727-79c6ce7c7167.png)
Any thoughts about?

TreeView itself is untouched for the time being.

**Comment by aNNiMON** [comment]
@mrEDitor yep, it's nice :+1: Btw, don't use black icons - the're highly contast with current UI. 

#515151
![image](https://cloud.githubusercontent.com/assets/1321838/9423829/33f52342-48dd-11e5-9dc7-9d7411f0b209.png)
